frontend/plugins/frontline_ui/src/modules/inbox/conversations/hooks/useConversations.tsx

```
@@ -9,23 +9,36 @@ import {
   const { list = [], totalCount = 0, pageInfo } = conversations || {};
+  const setNewMessagesCount = useSetAtom(newMessagesCountState);
+  const [refetchNewMessages, resetNewMessagesStates] = useAtom(
+    resetNewMessagesState,
+  );
+
+  useEffect(() => {
+    if (refetchNewMessages) {
+      refetch();
+      resetNewMessagesStates();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)